### DISCUSSION TITLE
Publish programming events on Github

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18945] [Other Feature Feedback, Questions, & Ideas] [answered]
#### 1. feature-name
Publish programming events on GitHub

#### 2. feature-description 
I would like to publish programming events on GitHub

#### 3. why
1. This allows GitHub to fund events for various open source communities.
2. This can make it easier for Github to make money from publicizing programming events from big tech companies, etc.
3. It is currently not possible to create or publish events on GitHub

**Comment by thuongtruong1009** [comment, marked as answer]
Hi @raphael-louis-andress 
Currently, the Github event only stagging in beta. If you want to publish with handwork and cooler. You can try with https://www.freecodecamp.org/news/publishing-github-event-data-with-github-actions-and-pages/

**Reply by ghost** [reply]
@thuongtruong1009 Hi! thank you for feedback ;D

**Reply by ghost** [reply]
Great feedback.

**Comment by rubikproxy** [comment]
**Feature Name:**  
Publish Programming Events on GitHub  

**Feature Description:**  
I would like to propose the ability to publish programming events directly on GitHub. This feature would allow users to create and share details about programming events such as hackathons, conferences, workshops, and meetups, making it easier for open-source communities and developers to stay informed and connected.  

**Why This Feature?**  
This feature can significantly enhance GitHub's value as a community platform:  
1. **Support Open Source Communities:** By funding and promoting programming events, GitHub can strengthen its role in nurturing open-source ecosystems.  
2. **Revenue Opportunity for GitHub:** Publicizing events from big tech companies and other sponsors could open new monetization channels for GitHub, enhancing its sustainability and reach.  
3. **Currently Missing Capability:** At present, GitHub lacks a dedicated feature for creating or publishing events, which limits its potential as a central hub for developer communities.
